src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
```

Comment:
placeholder should be bold on focus, I think previous mentor made a mistake
<img width="362" alt="image" src="https://user-images.githubusercontent.com/53611910/205526199-eaa407b0-af6f-47a3-9888-bade72b59c9d.png">